tests/integration/pure_cairo_vm/cheatcodes/expect_call/expect_call_test.cairo

```diff
@@ -10,6 +10,17 @@ func test_expect_call_success() {
   return ();
```

Comment:
we should add a test case for failing test when nothing was expected and an assertion was made